### DISCUSSION TITLE
Add failover alarm sound

### DISCUSSION THREAD

**Discussion by devdems** [LMS-Community/slimserver#851] [General]
Request to add failover/secondary alarm sound.

Use-case: when you have an alarm sound set to be a streaming radio. We should be able to choose secondary streaming radio if the primary source is temporarily down.

**Comment by michaelherger** [comment]
I'm sorry to say this is not going to happen. The alarm handling already is way too complicated (dealing with timeouts, missing network connectivity in case the Radio was asleep, to just name a few challenges). Adding another fallback before the final callback is not an option.

Can't you create a playlist with the two stations in it, and use that playlist as the alarm sound?

(converting this issue to a discussion instead)

**Reply by devdems** [reply]
I haven't thought about using a playlist for an alarm. Nice one! Will have a go at this option.
